# Lidar range value from 170째 - 190째 -> Caution median index -> 180째 => ix = [190,...181,180,179,177,...,170]
float32[] Lidar_Range_Values_At_180
# Lidar value at 180째
float32   Lidar_Value_At_180
# mean of range
float32   Mean